api/models/person.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (sequelize, DataTypes) => {
+  const Person = sequelize.define(
+    "Person",
+    {
+      identityId: DataTypes.STRING(100),
+      isBattlepassComplete: DataTypes.BOOLEAN
+    },
+    { tableName: "persons" }
+  );
+  Person.associate = function(models) {
+    // TODO: Add Contribution assocs here
```

Comment:
thoughts on associating Person w/ the HackerProfile? like a `Person.belongsTo(models.HackerProfile)`? this enables us to do something like `Person.getHackerProfile()` which is simpler than doing a `HackerProfile.where()` type of a call

Reply:
maybe instead of `belongsTo`, `hasOne` works better

Reply:
after looking at the PR this is a smaller nit now that i think about it, what's the logic again for having a separate persons table? just to differentiate participating hackers?

Reply:
Mostly just to separate concerns, so that application info sits in `hackerProfile`, and then all the points system has to worry about is the `persons` table. The idea is so that we can query all tables for anything with an id of the auth0 identity id, and that should identify a user across different components